src/controller/CopyFileNameController.ts

```diff
@@ -14,6 +21,18 @@ export class CopyFileNameController extends BaseCopyController {
         }
 
         const fileName = path.basename(sourcePath);
-        return copyAsync(fileName);
+        return copyAsync(fileName)
+        .catch((error: Error) => {
+            this.handleError(error.message);
```

Comment:
Error messages are usually handled in `extension.ts`. I would recommend to throw a new Error with updated message.